templates/app.yaml

```diff
@@ -86,7 +86,7 @@ Parameters:
   SolutionStackName:
     Type: String
     Description: The Beanstalk solution stack for the app
-    Default: '64bit Amazon Linux 2 v4.2.18 running Tomcat 8.5 Corretto 8'
+    Default: '64bit Amazon Linux 2 v4.9.5 running Tomcat 9 Corretto 8'
```

Comment:
The available version has changed to 4.9.8 :( 

https://docs.aws.amazon.com/elasticbeanstalk/latest/platforms/platform-history-java.html

I really wish we didn't have to include the exact version number here

Reply:
updated